packages/perseus-editor/src/components/color-circle.tsx

```diff
@@ -0,0 +1,41 @@
+import {View} from "@khanacademy/wonder-blocks-core";
+import {color as wbColor, spacing} from "@khanacademy/wonder-blocks-tokens";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+import type {LockedFigureColor} from "@khanacademy/perseus";
+
+type Props = {
+    color: LockedFigureColor;
+    filled?: boolean;
+};
+
+const ColorCircle = (props: Props) => {
+    const {color, filled = true} = props;
+
+    return (
+        <View
+            aria-label={`Color: ${color}, ${filled ? "filled" : "open"}`}
```

Comment:
What does having `aria-label` here do for us? Isn't this circle more of a decorative indicator next to the text of the option?

Reply:
Hmm, it's used both for both the color dropdown and for the overall summary section. That's a good point though, that would cause it to read twice in the dropdown. I guess I could add a `decorative` prop that causes the label to be added or not?